code/datums/status_effects/debuffs.dm

```diff
@@ -980,19 +980,21 @@
 	var/ants_remaining = 0
 
 /datum/status_effect/ants/on_creation(mob/living/new_owner, amount_left)
-	if(isnum(amount_left))
-		to_chat(new_owner, "<span class='userdanger'>You're covered in ants!</span>")
+	if(isnum(amount_left) && new_owner.stat >= HARD_CRIT)
+		if(new_owner.stat <= SOFT_CRIT) // Unconcious people won't get messages
+			to_chat(new_owner, "<span class='userdanger'>You're covered in ants!</span>")
 		ants_remaining += amount_left
 		RegisterSignal(new_owner, COMSIG_COMPONENT_CLEAN_ACT, .proc/ants_washed)
 	. = ..()
 
 /datum/status_effect/ants/refresh(effect, amount_left)
 	var/mob/living/carbon/human/victim = owner
-	if(isnum(amount_left) && ants_remaining >= 1)
-		if(!prob(1)) // 99%
-			to_chat(victim, "<span class='userdanger'>You're covered in MORE ants!</span>")
-		else // 1%
-			victim.say("AAHH! THIS SITUATION HAS ONLY BEEN MADE WORSE WITH THE ADDITION OF YET MORE ANTS!!", forced = /datum/status_effect/ants)
+	if(isnum(amount_left) && ants_remaining >= 1 && victim.stat >= HARD_CRIT)
+		if(victim.stat <= SOFT_CRIT) // Unconcious people won't get messages
```

Comment:
```
if victim.stat >= 3
   if victim.stat <= 1
```

?

Reply:
Oh good catch, that's supposed to have a !

Reply:
Wait no, I had it right the first time, that's correct.

Reply:
I just realized soft crit isn't unconscious, fixing that

Reply:
There we go, fixed it.

Reply:
```
if(victim.stat >= HARD_CRIT)
		if(victim.stat <= UNCONSCIOUS)
```
HARD_CRIT = 3
UNCONSCIOUS = 2

written out, this is
```
if(victim.stat is greater than or equal to 3)
		if(victim.stat is less than or equal to 2)    <----- you can't reach this is victim's is 3+
```